python/orca/example/torchmodel/train/mnist/main.py

```diff
@@ -69,16 +74,17 @@ def main():
     args = parser.parse_args()
 
     torch.manual_seed(args.seed)
-
+    if not exists(args.dir):
+        makedirs(args.dir)
     train_loader = torch.utils.data.DataLoader(
-        datasets.MNIST(args.dir, train=True, download=True,
+        datasets.MNIST(args.dir, train=True, download=False,
```

Comment:
User will fail to run this if he doesn't have mnist data downloaded.

Reply:
But jenkins failed if download = True.